packages/terra-data-grid/src/FlowsheetDataGrid.jsx

```diff
@@ -86,19 +95,42 @@ function FlowsheetDataGrid(props) {
     rowHeight,
     onCellSelect,
     onClearSelectedCells,
+    intl,
   } = props;
 
+  // Replace each non-header cell that contains no content with a dash indicating "No results".
```

Comment:
I think this comment exposes too much of the method implementation.  Also, it doesn't fully describe the current implementation or its future state.

Reply:
Removed, as the above comment explains it better anyway